src/book/mod.rs

```diff
@@ -519,6 +519,14 @@ impl MDBook {
         &[]
     }
 
+    pub fn uses_ace(&self) -> bool {
```

Comment:
Yeah serves me right for viewing the code on mobile while riding a bus ;) Disregard the comment sorry. Anyhow I would at least suggest to expose `PlaypenProperties` here instead of bool.